lib/ferrum/network.rb

```diff
@@ -59,12 +59,18 @@ def initialize(page)
     #   browser.network.wait_for_idle
     #
     def wait_for_idle(connections: 0, duration: 0.05, timeout: @page.browser.timeout)
+      puts "Ferrum: wait_for_idle: connections: #{connections} duration: #{duration} timeout: #{timeout}"
```

Comment:
Can we log all the requests, rather than as I read it above,  just the number of connections?